README.md

```diff
@@ -60,6 +60,24 @@ You can just drop the jar file in. It's a fat jar - it has all the dependencies
 
 [smtpapi-java.jar](https://sendgrid-open-source.s3.amazonaws.com/smtpapi-java/smtpapi-java.jar)
 
+## Setup Environment Variables
+
+### Initial Setup
+
+```bash
+cp .env_sample .env
```

Comment:
This step is not necessary if you do the next step. Basically, this would create a .env file that has the default data in it. 

I think it makes sense to remove this section and leave the "Environment Variable" section